dataland-frontend/src/components/forms/CreateEUTaxonomyForNonFinancials.vue

```diff
@@ -1,139 +1,452 @@
-      </template>
+          <template>
+            <SuccessUpload
+              v-if="postEuTaxonomyDataForNonFinancialsResponse"
+              msg="EU Taxonomy Data"
+              :message="postEuTaxonomyDataForNonFinancialsResponse.data"
+              :messageId="messageCount"
+            />
+            <FailedUpload v-else msg="EU Taxonomy Data" :messageId="messageCount" />
+          </template>
+        </div>
+        <JumpLinksSection :onThisPageLinks="onThisPageLinks" />
+      </div>
     </template>
   </Card>
 </template>
 
 <script lang="ts">
 import SuccessUpload from "@/components/messages/SuccessUpload.vue";
 import { FormKit } from "@formkit/vue";
+
+import Calendar from "primevue/calendar";
+import UploadFormHeader from "@/components/forms/parts/UploadFormHeader.vue";
+import PrimeButton from "primevue/button";
+
+import KPIfieldSet from "@/components/forms/parts/kpiSelection/KPIfieldSet.vue";
+
 import FailedUpload from "@/components/messages/FailedUpload.vue";
+import UploadReports from "@/components/forms/parts/UploadReports.vue";
+import BasicInformationFields from "@/components/forms/parts/BasicInformationFields.vue";
+
 import Card from "primevue/card";
 import { ApiClientProvider } from "@/services/ApiClients";
 import { humanizeString } from "@/utils/StringHumanizer";
-import DataPointFormElement from "@/components/forms/DataPointFormElement.vue";
 import { defineComponent, inject } from "vue";
+import { useRoute } from "vue-router";
 import Keycloak from "keycloak-js";
 import { assertDefined } from "@/utils/TypeScriptUtils";
+import { getHyphenatedDate } from "@/utils/DataFormatUtils";
+
+import {
+  euTaxonomyPseudoModelAndMappings,
+  euTaxonomyKpiInfoMappings,
+  euTaxonomyKpiNameMappings,
+} from "@/components/forms/parts/kpiSelection/EuTaxonomyPseudoModelAndMappings";
+import { CompanyAssociatedDataEuTaxonomyDataForNonFinancials } from "@clients/backend";
+import { UPLOAD_MAX_FILE_SIZE_IN_BYTES } from "@/utils/Constants";
+import { checkCustomInputs } from "@/utils/ValidationsUtils";
+import { modifyObjectKeys, objectType, updateObject } from "@/utils/UpdateObjectUtils";
+import { formatBytesUserFriendly } from "@/utils/NumberConversionUtils";
+import { ExtendedCompanyReport, ExtendedFile, WhichSetOfFiles } from "@/components/forms/Types";
+import JumpLinksSection from "@/components/forms/parts/JumpLinksSection.vue";
 
 export default defineComponent({
   name: "CreateEUTaxonomyForNonFinancials",
-  components: { DataPointFormElement, FailedUpload, Card, FormKit, SuccessUpload },
+  components: {
+    JumpLinksSection,
+    Calendar,
+    UploadFormHeader,
+    PrimeButton,
+    UploadReports,
+    BasicInformationFields,
+    KPIfieldSet,
+    FailedUpload,
+    Card,
+    FormKit,
+    SuccessUpload,
+  },
   setup() {
     return {
       getKeycloakPromise: inject<() => Promise<Keycloak>>("getKeycloakPromise"),
     };
   },
 
   data: () => ({
```

Comment:
Some things I have commented in the `data` object of `CreateEuTaxonomyFOrFinancials.vue` should also be addressed in the `data` object here. So whatever changes you do on those in `CreateEuTaxonomyFOrFinancials.vue`, please also do them here.